Error in migrateDirtyDBtoRealDB: "Cannot find module '../src/node_modules/ueberDB'"

Hello,

I am using etherpad-lite 1.6.0 with node 0.10.4.

I get this error when I run migrateDirtyDBtoRealDB.js:
```
Error: Cannot find module '../src/node_modules/ueberDB'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at EventEmitter.<anonymous> (/home/etherpad/etherpad-lite/bin/migrateDirtyDBtoRealDB.js:11:17)
    at process._tickCallback (node.js:415:13)
```

The correct path seems to be ```../src/node_modules/ueberdb2/```.

--- a/bin/migrateDirtyDBtoRealDB.js
+++ b/bin/migrateDirtyDBtoRealDB.js
@@ -7,8 +7,8 @@ require("ep_etherpad-lite/node_modules/npm").load({}, function(er,npm) {
   // file before using this script, just to be safe.
 
   var settings = require("ep_etherpad-lite/node/utils/Settings");
-  var dirty = require("dirty")('var/dirty.db');
-  var ueberDB = require("../src/node_modules/ueberDB");
+  var dirty = require("../src/node_modules/dirty")('var/dirty.db');
+  var ueberDB = require("../src/node_modules/ueberdb2");
   var log4js = require("../src/node_modules/log4js");
   var dbWrapperSettings = {
       "cache": "0",         // The cache slows things down when you're mostly writing.

